### ISSUE TITLE
Mögliche Features

### ISSUE THREAD

**Issue by obstschale** [obstschale/bookmark-manager#1]
Dieses Issue soll erstmal nur eine unsortierte Liste von möglichen und/oder gewünschten Features werden. Ob und wann etwas umgesetzt wird muss dann dediziert geklärt werden.
- Import / Export in verschiedene Formate (HTML, json, csv)

**Comment by wp-entwickler-at** [comment]
- bookmarklet
- autocompletion der tags/kategorien
- finden von bereits vorhandenen links
- taggen neu eingegebener links mit den gleichen tags, die bei gleichem domainnamen bereits verwenden wurden. zb golem.de/artikelx und golem.de/artikelxyz teilen sich ws die tags it oder pc oder tech. die dann vorschlagen
- speichern des link-contents (interessant ist ws nur der main-content, also ohne sidebar, werbung usw)
- speichern eines screenshots des linkziels

**Comment by Latz** [comment]
Nutzen von Meta-Tags. Zumindest "Yoast SEO" erzeugt einige Meta-Tags wie z.B. &lt;meta property="article:tag" /&gt; oder &lt;meta property="og:description" /&gt; die man direkt auswerten könnte. Ich weiß allerdings nicht, wie verbreitet diese Tags sind.

**Comment by websupporter** [comment]
Logisch:
- Suche nach Tags/Kategorien

Cool:
- Suche nach Domain/URL

Embeds ;)

**Comment by obstschale** [comment]
@Latz Die Meta Tags machen nur Sinn, wenn du die Links auch mit der `single.php` als eigenständigen Beitrag aufrufst.

**Comment by Latz** [comment]
@obstschale Nee, andersherum. Ich will die auslesen und automatisch im Formular einsetzen. Hatte ich ungünstig ausgedrückt.

**Comment by websupporter** [comment]
@Latz @obstschale 
Hatte das so verstanden. Gibst den Bookmark zum speichern ein, daraufhin wird die URL auf description etc. gecheckt und dann (falls es sowas gäbe) das Textfeld automatisch vorausgefüllt, damit ich mich nicht zu lange mit dem Speichern aufhalten muss.

Werden meta_keywords noch genutzt? Wären sonst vielleicht ganz interessant für Schlagworte. Oder ist das `article:tag` auch dafür

**Comment by obstschale** [comment]
Aso, dann habe ich das wirklich falsch verstanden. Alles klar 👍  Gute Idee.

**Comment by WebworkrNet** [comment]
Die Lesezeichen werden in Form eines Wordpress-Beitrags gespeichert?
Wenn ja, dann lässt ein Post bei Bedarf auf "privat" setzen?
Und falls das so ist, dann wäre diese Funktionalität auch bereits beim "Bookmarken" sinnvoll (Bookmarklet).

**Comment by obstschale** [comment]
@WebworkrNet Nur als Info: Ich habe deine Issues alle gesehen und auf dem Schirm. Aber aktuell habe ich wenig Zeit für das Projekt, daher gab es bis jetzt noch keine Antwort. Aber ich will weiter machen, sobald die Zeit es zu lässt :)
